src/test/all_browser.js

```diff
@@ -25,9 +25,8 @@ var testNames = [
     "ace/editor_navigation_test",
     "ace/editor_text_edit_test",
     "ace/editor_commands_test",
+    "ace/ext/command_bar_test",
     "ace/ext/hardwrap_test",
-    "ace/ext/inline_autocomplete_test",
```

Comment:
I see we still have "inline_autocomplete_test", is this a mistake removing it from all browser tests?

Reply:
I re-added it, along with some fixes to the tests there so they pass both in mockDOM and actual DOM.